app/controllers/organization.js

```diff
@@ -0,0 +1,123 @@
+const Organization = require('../models/Organisation')
+const handler = require('../utils/response-helper')
+const STATUS = require('../utils/status-codes')
+const helper = require('../utils/uploader')
```

Comment:
the variables should have same style.

Reply:
@devesh-verma I usually do UPPERCASE for constants and lowercase for others.
So should I change that?

Reply:
actually we are declaring everything as a constant which will make our lives miserable going forward.
As of now, handler and status always capital.
helper let it be like that.